### PR TITLE
Update doc

### PR DESCRIPTION
**Pull request by cn1036**
Add comments on available_date

### CODE REVIEW

**Review by doriapenciucgro — APPROVED** [review]

File: groclient/client.py [hunk 1]
```diff
@@ -1095,6 +1100,8 @@ def get_data_points(self, **selections):
         show_revisions : boolean, optional
             False by default, meaning only the latest value for each period. If true, will return
             all values for a given period, differentiated by the `reporting_date` field.
+        show_available_date : boolean, optional
```
Comment (muzigao) [inline groclient/client.py:1103]: @cn1036  could we also update output examples? I took out  those reference in previous PR https://github.com/gro-intelligence/api-client/pull/308/files

File: docs/data-point-definition.md [hunk 2]
```diff
@@ -8,6 +8,8 @@ Gro defines a "data point" as a discrete result produced by our API. When using
   'value': 0.131714797651,
   'unit_id': 851,
   'reporting_date': None,
+  'metadata': {},
+  'available_date': None,
```
Comment (jli) [inline docs/data-point-definition.md]: `None` shouldn't be a valid value of `available_date` anymore, right? It should either not appear in the result at all (if it wasn't requested), or it should have a timestamp string as the value?

**Comment by jli** [comment]
Is #301 still relevant, or can we close that PR?

File: docs/data-point-definition.md (hunk 2 above)
Comment (cn1036) [reply docs/data-point-definition.md]: good catch! updated with actual timestamp

**Comment by cn1036** [comment]
> Is #301 still relevant, or can we close that PR?

closed

**Review by jli — APPROVED** [review]

**Review by muzigao — CHANGES_REQUESTED** [review]

File: groclient/client.py (hunk 1 above)
Comment (muzigao) [reply groclient/client.py:1103]: ha never mind, since we removed available_date completely when it's not specified

**Review by muzigao — APPROVED** [review]

**Pull request merged**